### PR TITLE
Updates test tasks

### PR DESCRIPTION
**Pull request by lucasallan**

### CODE REVIEW

File: tasks/base.py [hunk 1]
```diff
@@ -7,8 +7,9 @@ class BaseTest(object):
     def download_mspec(self):
         if not os.path.isdir("../mspec"):
             run("cd .. && git clone --depth=100 --quiet https://github.com/rubyspec/mspec")
+            run("cd ../mespec && git checkout v1.6.0")
```
Comment (timfel) [inline tasks/base.py]: I'm pretty sure that's a typo

Comment (lucasallan) [inline tasks/base.py]: You mean the branch name? In this case, we're using the tag name.

Comment (kirbyfan64) [inline tasks/base.py]: @lucasallan I think @timfel is referring to `cd ../mespec`, which should probably be `cd ../mspec`.

Comment (lucasallan) [inline tasks/base.py]: oh I see, yes that's definitely a typo...

**Pull request merged**